libcudacxx/test/libcudacxx/std/strings/string.view/string.view.access/at.pass.cpp

```diff
@@ -23,6 +23,7 @@
 #include "literal.h"
 #include <nv/target>
 
+#if !_CCCL_COMPILER(NVRTC)
```

Comment:
This should rather be `TEST_HAS_EXCEPTIONS()`

Every code that uses `try` needs to be wrapped in that. We can also just drop the `__host__` there because it has no meaning